### PR TITLE
Add version to metastatus

### PR DESCRIPTION
**Pull request by mattmb**
I've wanted to find out the remote paasta version a few times so I've added it to metastatus here.

I've added a version endpoint in the API too for when we start using the API in our command line clients. @huadongliu can you check I haven't broken anything by moving the services endpoint?

### CODE REVIEW

File: paasta_tools/api/api_docs/swagger.json [hunk 1]
```diff
@@ -50,7 +64,7 @@
         ]
       }
     },
-    "/{service}/{instance}/status": {
+    "/services/{service}/{instance}/status": {
```
Comment (solarkennedy) [inline paasta_tools/api/api_docs/swagger.json]: Ah, I figured we would have to namespace this eventually

**Comment by solarkennedy** [comment]
I say ship. @huadongliu  should be primary though.

**Comment by nhandler** [comment]
I would also love to have an easy way to view the version of mesos/marathon/chronos running on individual hosts, but that would probably make things a bit cluttered. I guess we can simply continue to use `mco` for that.

Ship It!

**Comment by Rob-Johnson** [comment]
shipit

**Comment by huadongliu** [comment]
shipit

**Pull request merged**